README.md

```diff
@@ -65,30 +105,19 @@ Follow the instructions below to use the Thali Cordova Plugin on Android and iOS
 #### Android
 
 1. Add the Android platform
-   * cd ThaliTest
-   * cordova platform add android
+   * `cd ThaliTest`
+   * `cordova platform add android`
 2. Fix manifest min-sdk issue
-   * go to ThaliTest\platforms\android and in AndroidManifest.xml change android:minSdkVersion="10" to
-    android:minSdkVersion="16"
+   * go to `ThaliTest\platforms\android` and in `AndroidManifest.xml` change `android:minSdkVersion="10"` to
+    `android:minSdkVersion="16"`
 3. Add the plugin
-   * cordova plugin add https://github.com/thaliproject/Thali_Cordovaplugin
+   * `cordova plugin add https://github.com/thaliproject/Thali_Codovaplugin`
```

Comment:
You actually want that to say "Thali_Cordovaplugin", the missing 'r' is probably because you copied and pasted from an old Jukka checkin where he made that mistake. In fact he originally misnamed the entire project but we later fixed it. :)